api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
No, `strings.Replace` bails out if `old == new` (see: https://golang.org/src/strings/strings.go?s=17447:17493#L634).

I think it would be good to use `os.PathSeparator` so that this string is definitely correct and consistent with everything else on Windows.